lib/generator/app_localized_class_generator.dart

```diff
@@ -12,7 +12,7 @@ class AppLocalizedClassGenerator {
 switch (localeName) {
 ${canonicalizedLocales.map((it) => Intl.canonicalizedLocale(it.toLanguageTag())).map((it) => 'case "$it":\n  return new $it();'.indent(2)).join("\n")}
   default:
-    return null;
+    throw Exception('Could not find the locale: ' + localeName);
```

Comment:
@shana0440 Not sure about throwing an exception here, but obviously, we cannot return a null value.